person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_reader @id
+  attr_accessor @name, @age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] Include the `@id` here as well you can assign to a random number like: `@id = Random.rand(1..1000)`